source/skymaps/index.rst

```diff
@@ -84,7 +88,15 @@ Header Keywords
       lower and upper edges of each bin.  If there is a single element
       the column should be interpreted as the bin center.  For
       ``EBOUNDS`` or ``ENERGIES`` HDUs this keyword is optional.
-  
+
+* ``INTERP{IDX}``, type: string, **optional**
+    * Specify how the non-spatial bins are spaced and recommended to be interpolated.
+      Possible values are:
+        * ``"lin"``: linear spacing e.g. used for time or offset axes
+        * ``"log"``: logarithimic spacing e.g. used for energy axes
```

Comment:
Does it matter here if log is log10 or ln? log is ambiguous but I guess for interpolation it is not important

Reply:
In yesterdays aswg call, the need for some trigonometric scheme, e.g. 1/cos for zenith dependence was discussed

Reply:
No, for the interpolation the type of the log doesn't matter. I agree a trigonometric scheme is useful as well, but probably mostly for DL3. For the "skymaps" (DL4) the data is typically binned or project in sky coordinates already.

I think  currently the information on the bin spacing is not part of the "gadf" DL3 specification. E.g. in Gammapy we add it "by hand": https://github.com/gammapy/gammapy/blob/master/gammapy/irf/io.py#L5 In general it would be a useful addition!